src/app/Layout.tsx

```diff
@@ -9,13 +12,20 @@ interface LayoutProps {
 
 const Layout = (props: LayoutProps): null | JSX.Element => {
   return (
-    <>
+    <Box
+      id="app-main-layout"
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      sx={{
+        backgroundColor: Theme.colors.light.surface1,
+      }}>
```

Comment:
Can we move backgroundColor prop out of the sx object?

Reply:
Done